.github/workflows/update_sdk_methods.py

```diff
@@ -841,7 +841,7 @@ def parse(type, names):
                             ## and no app or robot interfaces do), then add the three inherited methods manually: Reconfigure(), DoCommand(), Close()
                             if '\tresource.Resource' in resource_interface.text:
                                 go_methods[type][resource]['Reconfigure'] = {'proto': 'Reconfigure', \
-                                    'description': 'Reconfigure must reconfigure the resource atomically and in place. If this cannot be guaranteed, then usage of AlwaysRebuild or TriviallyReconfigurable is permissible.', \
+                                    'description': 'Reconfigure must reconfigure the resource automatically and in place. If this cannot be guaranteed, then usage of AlwaysRebuild or TriviallyReconfigurable is permissible.', \
```

Comment:
I figured this was a typo? If not or if it was some other word lmk and I can change, I'm not sure

Reply:
nvm should have looked it up! did so and yes it is correct in the programming context lol